abekornelis/z390
# V1600next (#297)

* Pending changes for version v1.6.00 of z390 - not yet distributed

* fixing auto-merge mistakes

* V1600next upgrade to current code level (#295)

* Intermediate commit to trace moved and deleted files

* Reconstructed version v1.7.00 of z390 committed on behalf of Don Higgins

* Intermediate commit to trace renamed files

* Reconstructed version v1.7.01 of z390 committed on behalf of Don Higgins

* Reconstructed version v1.7.02 of z390 committed on behalf of Don Higgins

* Reconstructed version v1.7.03 of z390 committed on behalf of Don Higgins

* Reconstructed version v1.7.04 of z390 committed on behalf of Don Higgins

* Reconstructed version v1.7.05a of z390 committed on behalf of Don Higgins

* Reconstructed version v1.7.06 of z390 committed on behalf of Don Higgins

* Create README.md

* Moved files in preparation for V1.7.07

* Reconstructed version v1.7.07 of z390 committed on behalf of Don Higgins

* remove rpi directory and its files

* update z390.bat to use start z390.jar to release the command line

* rename v1707_readme.txt to README.TXT

* add demo missing files

* Add zpar missing files

* Add bash scripts

* Add zcobol/opt/CBLOPT.OPT; rename scripts

* Updated scripts

* Correct typos in comments

* Ensure jar directory exists

* ASMx.BAT enhancements

* CBLCx.BAT enhancements

* Linux Perl -> shell; fix z390 File->ASM.. bugs

* Allow program name relative path to begin with ..

* Add cblc, cblcl, cblclg to commands

* Add Z390.BAT to start GUI from terminal

* Add exit

* Allow terminal window closure after start GUI

* added 4 files to zcobol\demo, U*.* already there, skipped CLG.BAT, ZC_CALC.MAC not found

* 2 files and lower case dir zcobol\cvt added

* added zcobol\README.TXT

* add zcobol\i586 directory with  generated Intel assembler zcobol demo

* add zcobol\java directory with  generated native Java zcobol demo

* add zcobol\test dir

* add zcobol\vce\ proof of concept for zcobol Visual C generation

* add zpar missing files

* add 2 test files to assist\test

* add 2 missing files to vsam1\demo

* Issue 273: Rename TESTINS5.mlc to TESTINS5.MLC

* Issue 274 Four zopcheck directory file names should be all uppercase

* Issue #277 Program MMMMCIC3 should be retired

* Issue 279 Apply proper naming conventions to RT directory

* Issue #283 Apply proper naming conventions to SOA directory

* fixes for issue 281

* Issue #286 Apply proper naming conventions to cics directory

* Issue #289 Apply proper naming conventions - rename various files

* fixes for issue 282

* fixing auto-merge mistakes

Co-authored-by: Don Higgins <don@higgins.net>
Co-authored-by: Abe Kornelis <80408138+z390org@users.noreply.github.com>
Co-authored-by: dsh33782 <30640326+dsh33782@users.noreply.github.com>
Co-authored-by: John Ganci <jyganci@gmail.com>
Co-authored-by: John Ganci <58781735+jyganci@users.noreply.github.com>

* replacing ZDEFINE1.MAC with current version from main branch

* Fix version conflict on zcobol/mac/GEN_DIV.MAC

Co-authored-by: Don Higgins <don@higgins.net>
Co-authored-by: Abe Kornelis <80408138+z390org@users.noreply.github.com>
Co-authored-by: dsh33782 <30640326+dsh33782@users.noreply.github.com>
Co-authored-by: John Ganci <jyganci@gmail.com>
Co-authored-by: John Ganci <58781735+jyganci@users.noreply.github.com>
